bot/exts/github/github.py

```diff
@@ -0,0 +1,76 @@
+import typing
+
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from . import _issues, _profile, _source
+
+
+class Github(commands.Cog):
+    """Github."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @commands.group(name="github", aliases=("gh",), invoke_without_command=True)
+    async def github_group(self, ctx: commands.Context) -> None:
+        """Commands for Github."""
+        await ctx.send_help(ctx.command)
+
+    @github_group.command(name="profile")
+    @commands.cooldown(1, 10, BucketType.user)
+    async def profile(
+        self, ctx: commands.Context, username: typing.Optional[str]
+    ) -> None:
+        """
+        Fetches a user's GitHub information.
+
+        Username is optional and sends the help command if not specified.
+        """
+        if username is None:
+            raise commands.MissingRequiredArgument
+
+        github_profile = _profile.GithubInfo(self.bot.http_session)
+        embed = await github_profile.get_github_info(username)
+
+        await ctx.send(embed=embed)
+
+    @github_group.command(name="issue", aliases=("pr",))
+    async def issue(
+        self,
+        ctx: commands.Context,
+        numbers: commands.Greedy[int],
+        repository: typing.Optional[str] = None,
+        user: str = "gurkult",
```

Comment:
This could cause confusion when displaying usage.
```suggestion
        repository: typing.Optional[str] = None
```
You could do this, then do the following
1) if `repository` is `None`, `user = "gurkult"`
2. a) else `user, _, repository = repository.rpartition("/")`
2. b) if `user` is `""`, `user = gurkult`

Reply:
@Shivansh-007 what about this?

Reply:
Hmmm, I like the curren way, it is more understandable ig.

Reply:
the reason I don't like the current way is because the usage is technically `!issue <number> <repo> <user=''gurkult">` when we want it to be `!issue <number> <user=gurkult>/<repo>`

Reply:
and we can technically call `<user>/<repo>` as `<repo>`.